### PR TITLE
Multipart mime 3487 v4

### PR DESCRIPTION
**Pull request by catenacyber**
## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/3487

See https://github.com/OISF/suricata/pull/9424

#1377 with review taken into account
@inashivb do you approve ?

### CODE REVIEW

File: tests/mime/mime-dec-parse-full-msg-test01/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Test Description
+
+Test some mimre processing
+
+## PCAP
+
+Previous unit test for MIME in Suricata
```
Comment (inashivb) [inline tests/mime/mime-dec-parse-full-msg-test01/README.md:7]: They all say this but have a pcap within every test. Can we deduplicate it?
createst cannot do this at the moment. It copies pcap files to the test..

File: tests/mime/mime-dec-parse-full-msg-test01/test.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# *** Add configuration here ***
```
Comment (inashivb) [inline tests/mime/mime-dec-parse-full-msg-test01/test.yaml]: nit: These are indicators to add more conf like min version required and should/can be removed when no additional conf is needed

Comment (inashivb) [reply tests/mime/mime-dec-parse-full-msg-test01/test.yaml]: ah. i see its done in the next commit. I think these fixes should be a part of this commit.

**Review by inashivb — COMMENTED** [review]
Some nits inline.
I approve 💯 . Bring me more tests! 😄

File: tests/mime/mime-dec-parse-full-msg-test01/README.md (hunk 1 above)
Comment (catenacyber) [reply tests/mime/mime-dec-parse-full-msg-test01/README.md:7]: These are different unit tests / pcaps...
Do I miss something ?

Comment (jufajardini) [reply tests/mime/mime-dec-parse-full-msg-test01/README.md:7]: I'm assuming she thought they were all the same... 🤔  maybe just some rewording.
Maybe: `Adapted using data from a previous specific unit test for MIME in Suricata.` ?

Comment (catenacyber) [reply tests/mime/mime-dec-parse-full-msg-test01/README.md:7]: ok

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/1385

**Pull request closed without merging**